ironfish-cli/src/commands/wallet/multisig/dkg/round1.ts

```diff
@@ -13,14 +13,15 @@ export class DkgRound1Command extends IronfishCommand {
 
   static flags = {
     ...RemoteFlags,
-    secretName: Flags.string({
-      char: 's',
+    participantName: Flags.string({
+      char: 'n',
       description: 'The name of the secret to use for encryption during DKG',
+      aliases: ['name'],
     }),
     identity: Flags.string({
       char: 'i',
       description:
-        'The identity of the participants will generate the group keys (may be specified multiple times to add multiple participants). Must include the identity for secretName',
+        'The identity of the participants will generate the group keys (may be specified multiple times to add multiple participants). Must include the identity for participantName',
```

Comment:
(Does not need to be addressed here)

"Must include hte identity for participantName" << is this really true? Don't we add it automatically if omitted?

Reply:
Good catch! It's a small change, so I'll make it here